### DISCUSSION TITLE
V2 Feature expectations

### DISCUSSION THREAD

**Discussion by rigvedmaanas** [rigvedmaanas/CustomTkinterBuilder#5] [General]
Hello Everyone,

I would like to know what you guys want for the next update?

Please feel free to comment your feature requests

**Comment by sandeep24kt** [comment]


**Reply by rigvedmaanas** [reply]
Sure! I would be happy to help. But please post a new discussion in this [repo](https://github.com/rigvedmaanas/Realtime-System-Monitoring-Solution). This repository is related to Customtkinter Builder.

**Reply by sandeep24kt** [reply]
ok

**Comment by will-bug** [comment]
This project is awesome, looking forward to the online Video Player function!

**Comment by RM220507** [comment]
Thank you! I've needed something like this for so long!!

**Comment by federicomassi** [comment]
Great job, I can't wait to see grid support.
